bots/machine/machine_core/ssh_connection.py

```diff
@@ -233,7 +233,7 @@ def _ensure_ssh_master(self):
             self._start_ssh_master()
 
     def execute(self, command=None, script=None, input=None, environment={},
-                stdout=None, quiet=False, direct=False, timeout=120,
+                stdout=None, quiet=False, direct=False, timeout=120,raising=True,
```

Comment:
I don't see this being used anywhere? Please remove it again, it's much more explicit to either call a command with `|| true`, or even better, an explicit error handler.

Reply:
OK, Many thanks